src/components/PostPageDetail/index.js

```diff
@@ -691,7 +691,7 @@ const PostPageDetailIdComponent = (props) => {
                   itemParent={item}
                   comments={comments}
                   isLoading={loadingPost}
-                  refreshComment={handleRefreshComment}
+                  refreshComment={() => getComment(false, false)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the change in the `getComment` function parameters from `(false, true)` to `(false, false)` in the `refreshComment` prop may affect the functionality. Please ensure that this change is intended and does not break any existing functionality.
```diff
-    refreshComment={handleRefreshComment}
+    refreshComment={() => getComment(false, false)}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->